cms/articles/models.py

```diff
@@ -286,3 +314,10 @@ def previous_version(self, request: "HttpRequest", version: int) -> "TemplateRes
     def topic_ids(self) -> list[str]:
         """Returns a list of topic IDs associated with the parent article series page."""
         return list(self.get_parent().specific_deferred.topics.values_list("topic_id", flat=True))
+
+    @property
+    def figures_used_by_ancestor(self) -> list[str]:
```

Comment:
needs unit test

Reply:
Test added in [e564a03](https://github.com/ONSdigital/dis-wagtail/pull/152/commits/e564a03ec4ca4e78b3f6e330908e60572392685e)